AppController/djinn.rb

```diff
@@ -4085,6 +4085,22 @@ def build_uncommitted_changes()
     build_java_appserver if status.include?('AppServer_Java')
   end
 
+  def configure_ejabberd_cert
+    # Replace APPSCALE_CONFIG_DIR/ejabberd.pem with private key and cert
+    # copied from tools. APPSCALE_CONFIG_DIR/ejabberd.pem is not the default
```

Comment:
I would phrase this as 'belonging to the deployment.' or something similar, instead of 'copied from tools.' since I'm not sure the tools generating it is the best approach and we may change it.